Formula/aften.rb

```diff
@@ -32,6 +32,10 @@ class Aften < Formula
     sha256 "949dd8ef74db1793ac6174b8d38b1c8e4c4e10fb3ffe7a15b4941fa0f1fbdc20"
   end
 
+  # The ToT actually compiles fine, but there's no official release made from that changeset.
+  # So fix the Apple Silicon compile issues.
+  patch :DATA
```

Comment:
This should be submitted upstream, or if it's there already upstream should be asked for a release.

Reply:
Or we should apply a specific commit from ToT.

Reply:
Alright I have done as you requested and applied a specific commit from ToT. Unfortunately the project itself seems to be abandonware, the last commit was in 2011, but the currently packaged version is from 2007. People have posted patches to the mailing list but no one replies to any e-mails there.